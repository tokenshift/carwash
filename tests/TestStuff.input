This is a test.
Config:
PASSWORD=GABBAgabbaHey
DATABASE_URL=postgres://DbUser:DbPassword@DbHost:5432/Testing
Doing some more stuff
Just make sure nobody knows the gabbagabbahey password!
Or the DBPASSWORD. That would be bad.
ENV password rocketx
ENV password=bigred
ENV password=flash
	secret_key=another_thing
	api_token=dontsharethis
	other_config=not-secret
{"stream":"ENV MY_PASSWORD some_secret_thing\n"}
