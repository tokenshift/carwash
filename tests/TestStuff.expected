This is a test.
Config:
PASSWORD=********
DATABASE_URL=postgres://DbUser:********@DbHost:5432/Testing
Doing some more stuff
Just make sure nobody knows the ******** password!
Or the ********. That would be bad.
ENV password ********
ENV password=********
ENV password=********
	secret_key=********
	api_token=********
	other_config=not-secret
{"stream":"ENV MY_PASSWORD ********\n"}
